packages/experiments/src/components/Tile/Tile.tsx

```diff
@@ -29,7 +29,7 @@ export interface ITileState {
   isModal?: boolean;
 }
 
-const SIZES: {
+export const SIZES: {
```

Comment:
instead of exporting "SIZES", can you rename this to something specific to tile, and use TitleCasing?

TileLayoutSizes would be reasonable.

Reply:
Think of it like this: When you `import {} from '@uifabric/experiments';` the intellisense will suggest importing `SIZES`. Does that make sense to the caller?

Reply:
Got it. Thanks for the feedback. I guess if you are putting it in the context of intellisense when importing makes total sense.